app/views/user_mailer/admin_deleted_work_notification.html.erb

```diff
@@ -1,11 +1,11 @@
 <% content_for :message do %>
-  <p>Dear <%= @user.default_pseud.byline %>,</p>
+  <p><%= t 'user_mailer.admin_deleted_work_notification.hello', user: @user.default_pseud.byline %></p>
   
-  <p>Your work <i><%= style_bold(@work.title.html_safe) %></i> was deleted from the Archive by a site admin.</p>
+  <p><%= t('user_mailer.admin_deleted_work_notification.html.part1', title: style_bold(@work.title.html_safe)).html_safe %>
   
-  <p>If your work was part of an import project managed by our Open Doors team, please <%= opendoors_link("contact Open Doors") %> with any further questions.</p>
+  <p><%= t('user_mailer.admin_deleted_work_notification.html.part2', opendoors_link: opendoors_link(t 'user_mailer.admin_deleted_work_notification.opendoors')).html_safe %>
  
-  <p>If it's possible your work violated the Archive's Terms of Service, please <%= abuse_link("contact our Abuse Committee") %>.</p>
+  <p><%= t('user_mailer.admin_deleted_work_notification.html.part3', abuse_link: abuse_link(t 'user_mailer.admin_deleted_work_notification.abuse_committee' ) ).html_safe %>
```

Comment:
There are some extra spaces inside the parentheses on the end of this line: 

`(t 'user_mailer.admin_deleted_work_notification.abuse_committee' ) ).html_safe` 

It should be:

`(t 'user_mailer.admin_deleted_work_notification.abuse_committee')).html_safe`